biogeochem/FatesAllometryMod.F90

```diff
@@ -2056,6 +2058,18 @@ subroutine set_root_fraction(root_fraction, ft, zi, lowerb, icontext )
        call endrun(msg=errMsg(sourcefile, __LINE__))
     end select
 
+!    if( abs(sum(root_fraction)-1.0_r8) > 1.e-9_r8 ) then
```

Comment:
Flagging this for future discussion @rgknox.  I'm wondering if we should have some sort of check or test for corrections.